### PR TITLE
add domain tn5250j.org to the website

### PR DESCRIPTION
**Pull request by cycloon**
I just registered tn5250j.org for the project. Now the package names again make (more) sense. Add this file to enable the domain on Github Pages.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+tn5250.org
```
Comment (duncan3dc) [inline CNAME:1]: Surely this should be `tn5250j.org`?

**Comment by duncan3dc** [comment]
This pr broke the website, `tn5250j.github.io` redirects to `tn5250.org` which isn't registered

**Comment by cycloon** [comment]
Sorry for messing things up. Of course this was a typo and meant to be tn5250j.org. Also it seems that the .org zone is not updated everywhere all 15 mins as it should.

**Comment by duncan3dc** [comment]
@cycloon No problem, :hankey: happens. I normally leave PRs for @nitram509 to merge

**Comment by cycloon** [comment]
after #4 was merged, it works now. does it for you?

**Comment by duncan3dc** [comment]
Yes, `tn5250j.org` is showing the website for me now

**Comment by cycloon** [comment]
but tn5250j.github.io still redirects to tn5250.org. Some caching seems to still interfere

**Comment by duncan3dc** [comment]
Yeh, that's not much of a big deal though, only those of us that got the dodgy CNAME record will be affected (probably just me and you).

This change does mean we've lost support for SSL though

**Comment by cycloon** [comment]
there is no way to support ssl on custom domains - right?

**Comment by duncan3dc** [comment]
I've never done it myself, but others appear to have had success:
https://sheharyar.me/blog/free-ssl-for-github-pages-with-custom-domains/

**Comment by cycloon** [comment]
What weights more for you? Top level domain or https?

**Comment by duncan3dc** [comment]
Personally for me it would be SSL, I'd just have `http://tn5250j.org` redirect to `https://tn5250j.github.io`

**Comment by cycloon** [comment]
I can do that too if you want. Sorry if I was too quick with all that. Was too tempting to click the merge button. Will wait for another member next time.

**Comment by duncan3dc** [comment]
Let's wait for @nitram509

**Comment by nitram509** [comment]
Wow, how cool is that, our own top-level domain :)
👍 

Preferable I would also vote for SSL support. 
But considering we don't have any sessions or other tokens to steal, I could live without SSL.
If it's just about automatic redirect, we could "refactor" external links as much as possible. This would reduce the chance, somebody would go without SSL.

Other views on that topic?

**Pull request merged**